### DISCUSSION TITLE
How to style `next/link` in Next.js 13

### DISCUSSION THREAD

**Discussion by ephraimduncan** [vercel/next.js#41819] [Help] [answered]
In previous versions, you could style the `<a>` tag to affect the Link component. With Next.js 13, The `next/link` child can no longer be `<a>`. How ___should___ styling of Link in Next.js 13 be done?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

In Next 12.x, you could already use the new link behavior by passing `legacyBehavior={false}`. Now in Next 13, `legacyBehavior` is false, although I think the TS docs are wrong. I'll do a PR for that.

To style the `a` tag , just pass a className prop.

```jsx
     <Link href="/" className="foo">
        home
      </Link>
```

or 

```jsx
      <Link href="/" className="foo">
        <span>home</span>
      </Link>
```

More details from the time this was proposed initially, https://github.com/vercel/next.js/pull/36436:

> When using the new behavior props that can be passed to <a> can be passed to <Link>. Previously you could do something like <Link href="/somewhere"><a target="_blank">Download</a></Link> but with <Link> rendering <a> it now allows these props to be set on link. E.g. <Link href="/somewhere" target="_blank"></Link> / <Link href="/somewhere" className="link"></Link>

And to avoid confusion:

> This default will be changed in Next.js 13, it does not affect existing apps in Next.js 12 unless opted in via experimental.newNextLinkBehavior and running the codemod.

Since v9.5 IIRC, passing a string to Link was also accepted, but not very used. https://nextjs.org/docs/tag/v9.5.2/api-reference/next/link#using-a-component-that-supports-onclick
